### DISCUSSION TITLE
wallet standard broken link

### DISCUSSION THREAD

**Discussion by gftea** [aptos-labs/aptos-developer-discussions#234] [Dev Doc and Tutorial Feedback]
the [wallet standard link ](https://aptos.dev/guides/wallet-standard) in react adaptor package https://www.npmjs.com/package/@aptos-labs/wallet-adapter-react  is broken

**Comment by Alexzander-Stone** [comment]
Thanks for the heads up. 😌

Sent a [PR](https://github.com/aptos-labs/aptos-wallet-adapter/pull/288) to the team to update the link.

**Comment by Alexzander-Stone** [comment]
PR to fix dead link is merged.
